CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.10)
+
+project(ExaMiniMD LANGUAGES CXX)
+
+find_package(Kokkos REQUIRED)
+option(USE_MPI "MPI" ON)
+
+if (USE_MPI)
+  add_definitions(-DEXAMIND_ENABLE_MPI)
```

Comment:
Remove in favor of line below.

Reply:
51cd06621d17b6a165f9a3545355ce8a9bcd01f8